### PR TITLE
Added citation related infos to mqc module and overview file

### PR DESCRIPTION
**Pull request by hkimaf**
I have added the citation information for JCC(2021) paper in MQC module and PyUNIxMD manual.

### CODE REVIEW

File: docs/source/overview.rst [hunk 1]
```diff
@@ -42,6 +42,13 @@ Authors
 ---------------------------
 The current version of PyUNIxMD has been developed by Seung Kyu Min, In Seong Lee, Jong-Kwon Ha, Daeho Han, Kicheol Kim, Tae In Kim, Sung Wook Moon in the Theoretical/Computational Chemistry Group for Excited State Phenomena of Ulsan National Institute of Science and Technology (UNIST). 
 
+
+Citation
+---------------------------
+Please cite the following work when publishing results from PyUNIxMD program:
+
+Lee, I. S., Ha, J. K., Han, D., Kim, T. I., Moon, S. W., & Min, S. K. (2021). PyUNIxMD: A Python‐based excited state molecular dynamics package. Journal of computational chemistry, 42(24), 1755-1766.
```
Comment (jkha-unist) [inline docs/source/overview.rst:50]: Hyphen is missing in "J. K.". It should be "J.-K.".

Same for print_init in mqc.py.

Comment (jkha-unist) [reply docs/source/overview.rst:50]: Plus, I think it should be "Journal of **C**omputational **C**hemistry" with capital letters.

**Comment by inseonglee** [comment]
Please, show compiled manual.

File: src/mqc/mqc.py [hunk 2]
```diff
@@ -246,7 +246,9 @@ def print_init(self, qm, mm, restart):
         {"-" * 68}
 
         {" " * 4}Please cite PyUNIxMD as follows:
-        {" " * 4}This is article
+        {" " * 4}Lee, I. S., Ha, J. K., Han, D., Kim, T. I., Moon, S. W.,
+        {" " * 4}& Min, S. K. (2021). PyUNIxMD: A Python‐based excited state molecular dynamics package.
```
Comment (inseonglee) [inline src/mqc/mqc.py:250]: This line is too long. It is much better to move 'molecular dynamics package' sentence below.

**Comment by inseonglee** [comment]
Overall, I think the order (First name and last name) would be better since the developer lists show this order.

**Comment by inseonglee** [comment]
Please, use italic letter for Journal in manual.

**Comment by inseonglee** [comment]
@jkha-unist @DaehoHan @tikimtikim @swmoon-unist
For citations, should we use identical citation style as used in reference section?
Current manual uses 'unsrt' and other default settings in bibtex.

**Comment by DaehoHan** [comment]
I'd like to keep this version and change the citation style into ACS later.

**Comment by inseonglee** [comment]
OK

**Pull request merged**